vms/avm/service_test.go

```diff
@@ -1782,14 +2014,20 @@ func buildSecpMintOp(createAssetTx *txs.Tx, key *secp256k1.PrivateKey, outputInd
 	}
 }
 
-func buildOperationTxWithOp(chainID ids.ID, op ...*txs.Operation) *txs.Tx {
-	return &txs.Tx{Unsigned: &txs.OperationTx{
-		BaseTx: txs.BaseTx{BaseTx: avax.BaseTx{
-			NetworkID:    constants.UnitTestID,
-			BlockchainID: chainID,
-		}},
-		Ops: op,
-	}}
+func buildOperationTxWithOp(t *testing.T, env *environment, ops []*txs.Operation) *txs.Tx {
```

Comment:
1. Since ops is a slice seems to make sense to name it `WithOps` instead of `WithOp`
2. Did we change the signature from optional parameter to required because we don't want to create support creating txs without the `Ops` field which would be considered invalid?
```suggestion
func buildOperationTxWithOps(t *testing.T, env *environment, ops []*txs.Operation) *txs.Tx {
```